src/rustllvm/RustWrapper.cpp

```diff
@@ -469,12 +474,17 @@ extern "C" LLVMRustMetadataRef LLVMRustDIBuilderCreateStructType(
     unsigned LineNumber,
     uint64_t SizeInBits,
     uint64_t AlignInBits,
-    unsigned Flags,
     LLVMRustMetadataRef DerivedFrom,
     LLVMRustMetadataRef Elements,
     unsigned RunTimeLang,
     LLVMRustMetadataRef VTableHolder,
     const char *UniqueId) {
+#if LLVM_VERSION_GE(4, 0)
+  auto Flags = DINode::FlagZero;
+#else
+  unsigned Flags = 0;
```

Comment:
All the `DINode::FlagFoo` names seem to exist in LLVM 3.7 so why not just use those instead of magic numbers?